docs/zh_cn/04-developer-guide/do_regression_test.md

```diff
@@ -49,56 +68,54 @@ python ./tools/regression_test.py \
 
 ## 例子
 
-1. 测试 mmdet 和 mmpose 的所有 backend 的 转换+精度
+1. 测试 mmdet 和 mmpose 的所有 backend 的 **转换+精度**
 
 ```shell
 python ./tools/regression_test.py \
-    --deploy-yml ./configs/mmdet/mmdet_regression_test.yaml ./configs/mmpose/mmpose_regression_test.yaml \
+    --codebase mmdet mmpose \
     --backends all \
     --work-dir "../mmdeploy_regression_working_dir" \
     --device "cuda" \
     --log-level INFO \
     --performance
 ```
 
-2. 测试 mmdet 和 mmpose 的某几个 backend 的 转换+精度
+2. 测试 mmdet 和 mmpose 的某几个 backend 的 **转换+精度**
 
 ```shell
 python ./tools/regression_test.py \
-    --deploy-yml ./configs/mmdet/mmdet_regression_test.yaml ./configs/mmdet/mmpose.yaml \
+    --codebase mmdet mmpose \
     --backends onnxruntime tesnsorrt \
     --work-dir "../mmdeploy_regression_working_dir" \
     --device "cuda" \
     --log-level INFO \
-    --performance
+    -p
 ```
 
-3. 测试 mmdet 和 mmpose 的某几个 backend，只需测试转换
+3. 测试 mmdet 和 mmpose 的某几个 backend，**只测试转换**
 
 ```shell
 python ./tools/regression_test.py \
-    --deploy-yml ./configs/mmdet/mmdet_regression_test.yaml ./configs/mmdet/mmpose.yaml \
+    --codebase mmdet mmpose \
     --backends onnxruntime tesnsorrt \
     --work-dir "../mmdeploy_regression_working_dir" \
     --device "cuda" \
     --log-level INFO
 ```
 
-## 2. 回归测试配置文件
+## 3. 回归测试配置文件
 
 ### 示例及参数解析
 
 ```yaml
 globals:
-  codebase_name: mmocr # 回归测试的 codebase 名称
   codebase_dir: ../mmocr # 回归测试的 codebase 路径
   checkpoint_force_download: False # 回归测试是否重新下载模型即使其已经存在
-  checkpoint_dir: ../mmdeploy_checkpoints # 回归测试是否下载模型的路径
   images: # 测试使用图片
-    img_224x224: &img_224x224 ./tests/data/tiger.jpeg
-    img_300x300: &img_300x300
-    img_800x1344: &img_cityscapes_800x1344
-    img_blank: &img_blank
+    img_densetext_det: &img_densetext_det ../mmocr/demo/demo_densetext_det.jpg
```

Comment:
Removing `img_224x224` affects the following `convert_image`, doesn't it?
```yaml
  convert_image: &convert_image # 转换会使用到的图片
    input_img: *img_224x224
    test_img: *img_300x300
```

Reply:
Fixed